ui/oopsyraidsy/buff_map.ts

```diff
@@ -47,51 +50,136 @@ export const missedEffectBuffMap: readonly MissableEffect[] = [
     effectId: 'A26',
     collectSeconds: 2,
   },
+  {
+    // BRD
+    id: 'Mage\'s Ballad',
+    type: 'damage',
+    effectId: '8A9',
+    collectSeconds: 45,
+  },
+  {
+    // BRD
+    id: 'Army\'s Paeon',
+    type: 'damage',
+    effectId: '8AA',
+    collectSeconds: 45,
+  },
+  {
+    // BRD
+    id: 'The Wanderer\'s Minuet',
+    type: 'damage',
+    effectId: '8A8',
+    collectSeconds: 45,
+  },
```

Comment:
```suggestion
```

I would skip these.  There used to be a comment about it but I think that comment got lost when these abilities moved into a different file.

My recollection is that they pulse every three seconds, and it's very noisy when people are being missed.  Collecting for 45 seconds is "did somebody get hit at all during that time period" which isn't right either.